clients/src/test/java/org/apache/kafka/clients/producer/internals/TransactionManagerTest.java

```diff
@@ -2449,7 +2465,7 @@ private void doInitTransactions(long pid, short epoch) {
 
     private void assertAbortableError(Class<? extends RuntimeException> cause) {
         try {
-            transactionManager.beginTransaction();
+            transactionManager.beginCommit();
```

Comment:
Is this relevant to the PR?

Reply:
Not directly, but this seemed like a better way to verify an abortable error. We expect `beginTransaction()` to fail regardless whether the error is abortable or fatal.